### PR TITLE
Ay/collection metadata

### PR DESCRIPTION
**Pull request by ayang-rkjs**

### CODE REVIEW

File: src/api/nft.ts [hunk 1]
```diff
@@ -45,6 +46,28 @@ export interface NftContractForNft extends NftContract {
...
 }
 
+/**
+ * Alchemy representation of an NFT collection
+ *
+ * @public
+ */
+export interface NftCollection {
+  /** The name of the collection. */
+  name?: string;
```
Comment (thebrianchen) [inline src/api/nft.ts:56]: name should always be present

**Review by thebrianchen — COMMENTED** [review]
ty! mainly concerned about the nullability/undefined of some of the fields

**Review by thebrianchen — APPROVED** [review]
lgtm!

File: src/util/util.ts [hunk 2]
```diff
@@ -78,6 +85,13 @@ export function getNftContractFromRaw(
   });
 }
 
+export function getNftCollectionFromRaw(
+  rawNftCollection: RawNftCollection
+): NftCollection {
+  // No transformation needed.
+  return rawNftCollection;
```
[the remark this answers is not in the document]
Comment (ayang-rkjs) [reply src/util/util.ts]: ah very nice

**Pull request merged**